runtime/autoload/provider/node.vim

```diff
@@ -71,13 +71,15 @@ function! provider#node#Detect() abort
     let yarn_opts = deepcopy(s:NodeHandler)
     let yarn_opts.entry_point = '/node_modules/neovim/bin/cli.js'
     " `yarn global dir` is slow (> 250ms), try the default path first
-    " XXX: The following code is not portable
-    " https://github.com/yarnpkg/yarn/issues/2049#issuecomment-263183768
+    " Provides support for Yarn node_modules on Win32
```

Comment:
This comment looks unnecessary.
```suggestion
```

Reply:
Thanks, I'll leave the original link as reference.